knowledge-base/grid-custom-filter-menu.md

```diff
@@ -0,0 +1,201 @@
+---
+title: How to Refresh Filter Menu Context After Programmatic Changes
+description: Learn how to refresh filter menu context after programmatic changes with a custom component
+type: how-to
+page_title: How to Refresh Filter Menu Context After Programmatic Changes
+slug: grid-kb-custom-filter-menu
+tags: grid, blazor, custom, filter, filtermenu
```

Comment:
Tags should make sense outside of context. See the forums tags in the right column here: https://www.telerik.com/forums/blazor and imagine you see a "custom" tag there.

```suggestion
tags: blazor, grid, filter, filtermenu
```